OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -228,6 +231,8 @@ public MinefieldOrderGenerator(Actor a, CPos xy, bool queued)
 					tileBlocked = a.World.Map.Rules.Sequences.GetSequence("overlay", "{0}-{1}".F(minelayer.Info.TileInvalidName, tileset)).GetSprite(0);
 				else
 					tileBlocked = a.World.Map.Rules.Sequences.GetSequence("overlay", minelayer.Info.TileInvalidName).GetSprite(0);
+
+				cursor = minelayer.Info.DeployCursor;
```

Comment:
This is the wrong cursor type!

Reply:
Ah, yes that happened during all the rebases.